packages/launcher/test/unit/detect_spec.ts

```diff
@@ -141,5 +141,39 @@ describe('browser detection', () => {
         warning: firefoxGcWarning,
       })
     })
+
+    it('works with :browserName format', async () => {
+      detectByPath('/foo/bar:foo-browser').then((browser) => {
```

Comment:
this won't be tested as expected since the promise is not `await`ed, though it is an async function